scipy/spatial/distance.py

```diff
@@ -931,6 +933,57 @@ def kulsinski(u, v, w=None):
     return (ntf + nft - ntt + n) / (ntf + nft + n)
 
 
+def kulczynski1(u, v, w=None):
+    """
+    Compute the Kulczynski 1 dissimilarity between two boolean 1-D arrays.
+
+    The Kulczynski 1 dissimilarity between two boolean 1-D arrays `u` and `v`,
+    is defined as
+
+    .. math::
+
+         \\frac{c_{TT}}
+              {c_{FT} + c_{TF}}
```

Comment:
What is TT, FT, TF? It does not relate to `c_{ij}`

Reply:
The documentation is also following the same pattern, `hamming`, `jaccard`, etc. Should we make the change there too?

Reply:
Let's get this first right here and also contained. You/we can then make a followup which addresses just the documentation of similar functions. Also Hamming seems to be better as does not introduce more notations (I am not saying this is the best OC. It's still missing a definition for `k` for instance).